src/Microsoft.IdentityModel.Clients.ActiveDirectory/Internal/Helpers/AdalIdHelper.cs

```diff
@@ -67,6 +67,8 @@ internal static class AdalIdParameter
     /// </summary>
     internal static class AdalIdHelper
     {
+        internal static Regex AdalVersionRegex = new Regex(@"Version=[\d]+.[\d+]+.[\d]+.[\d]+", RegexOptions.CultureInvariant);
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Change the visibility of 'AdalVersionRegex' or make it 'const' or 'readonly'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://iddp-sonarqube:9000/coding_rules#rule_key=csharpsquid%3AS2223)

Reply:
make it private

Reply:
done.